install/ubuntu-installation-guide.md

```diff
@@ -0,0 +1,55 @@
+# Installing Kata Containers on Ubuntu
+
+Note:
+Kata Containers is available for Ubuntu\* **16.04** and **17.10**.
+Currently Kata Containers is currently only build for x86_64.
+
+
+This step is only required in case Docker is not installed on the system.
+1. Install the latest version of Docker with the following commands:
+
+```
+$ sudo -E apt-get -y install apt-transport-https ca-certificates wget software-properties-common
+$ wget -qO - https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+$ sudo -E add-apt-repository "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
This does give a warning about the docker site not providing i386 binaries. Hence, I think we're going to have to add the `[arch=...]` value as we did for Clear Containers:

- https://github.com/clearcontainers/runtime/blob/master/docs/ubuntu-installation-guide.md

Since we don't want to hardcode the architecture and since `arch(1)` doesn't dtrt, you can instead do this:

```
$ arch=$(dpkg --print-architecture)
$ sudo -E add-apt-repository "deb [arch=${arch}] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```